VideoLocker/res/layout/row_discussion_topic.xml

```diff
@@ -1,19 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical"
+<org.edx.mobile.view.custom.ETextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_topic_name_text_view"
+    style="@style/regular_grey_text"
     android:layout_width="match_parent"
-    android:layout_height="50dp">
-
-    <org.edx.mobile.view.custom.ETextView
-        android:id="@+id/discussion_topic_name_text_view"
-        style="@style/regular_grey_text"
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
-        android:gravity="center_vertical"
-        android:paddingLeft="30dp"
-        android:ellipsize="end"
-        android:singleLine="true"
-        android:textDirection="locale"
-        android:textSize="16sp" />
-
-</LinearLayout>
\ No newline at end of file
+    android:layout_height="wrap_content"
+    android:ellipsize="end"
+    android:gravity="center_vertical"
+    android:padding="@dimen/edx_margin"
+    android:singleLine="true"
+    android:textDirection="locale"
+    android:textSize="16sp"
+    tools:text="Topic Name" />
```

Comment:
This seems to be mostly the same as the `TextView` items used in the header. Maybe there should be a generic style created for this list item? Or perhaps this layout could be reused using the `include` tag?